### DISCUSSION TITLE
Welcome to rules_license Discussions!

### DISCUSSION THREAD

**Discussion by aiuto** [bazelbuild/rules_license#76] [Announcements]
## Welcome

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * This augments general list at [bazel-ssc@bazel.build](https://groups.google.com/a/bazel.build/g/bazel-ssc)

**Comment by mzeren-vmw** [comment]
👋

**Comment by danielmachlab** [comment]
Cool!
